src/main/java/model/item/Weapon.java

```diff
@@ -24,11 +55,19 @@ public void setDice(String dice) {
     }
 
     public int getDamage() {
+        return damage.get();
+    }
+
+    public IntegerProperty damageProperty() {
         return damage;
     }
 
     public void setDamage(int damage) {
-        this.damage = damage;
+        this.damage.set(damage);
+    }
+
+    public IntegerBinding damageWithWearBinding() {
```

Comment:
the decorator pattern would make this 
`getDamage()`
`getDamageWithWear()`
`getDamageWithUpgradesAndWearAtMoonlight()`
stuff obsolete.

Reply:
What do you mean exactly here?

Reply:
we talked about this

Reply:
The same should be done with the protection of armor.
A good and general solution would be best here. 

Opened own feature request.